packages/contentful--app-scripts/lib/upload/create-zip-from-directory.js

```diff
@@ -6,9 +6,13 @@ async function createZipFileFromDirectory(path) {
   try {
     const zip = new AdmZip();
     zip.addLocalFolder(path);
-    console.log('----------------------------');
-    console.log(`${chalk.yellow('Done!')} Files from ${chalk.dim(path)} successfully zipped.`);
-    console.log('----------------------------');
+    console.log(`
+  ----------------------------
+
```

Comment:
very nit: in the logs above we don't have the blank lines between `----`s 😬

Reply:
you mean an extra line break was added by mistake? If so, it was to make sure the logs look better 🤷‍♂️ 
![Screenshot 2021-04-21 at 12 30 29](https://user-images.githubusercontent.com/3318312/115539598-74519180-a29d-11eb-9e15-24bfd9813936.png)